### ISSUE TITLE
UZBL with omxplayer

### ISSUE THREAD

**Issue by matteoraggi** [uzbl/uzbl#329]
Please paste the output of `uzbl-core --bug-info` here:
uzbl-event-manager:ERROR: event manger already started with pid 1235
```
Hi, I am reading in many forums online that  uzbl is showing videos quickly thanks to OMXPLAYER, but when I launch a mp4/h264 video file linke this
https://graftechnology.com/video/
It show the video scattering. It it slow like chromium, firefox and all other the raspberry browsers by default.
I am using Raspberry pi 3 model B and raspbian 8.0 and uzbl last version. 
Please help!
I just would like to show slideshows of images with a videos sometimes, but the videos actually are very poorly played from raspberry...

**Comment by mathstuf** [comment]
Looks like you're using an old uzbl; could you update to `master`?

**Comment by matteoraggi** [comment]
from the ADD/REMOVE software of raspbian 8.0 I read that I am using uzbl-0.0.0.0git 20120514-1.1+b1.
is that ok? or where or how I can update my uzbl?

**Comment by mathstuf** [comment]
Building from source isn't bad if webkit headers are readily available (should be in Raspian).

I'm assuming that omxplayer is a plugin thing (like flash and friends?).

**Comment by matteoraggi** [comment]
I uninstalled uzbl of mayb 2012 from the ADD/REMOVE SOFTWARE and then I installed back a new fresh version of uzbl writing
`sudo apt-get install uzbl`
and it download 1.6mb!
but I still read the same old version: " uzbl-0.0.0.0git 20120514-1.1+b1"
so then I tryed to update everything with 
`sudo apt-get update`
and
`sudo apt-get upgrade`
but raspbian can't find updates and it still shows the old 212 may version..
any suggestions?

**Comment by matteoraggi** [comment]
omxplayer is a standalone player and I was reading in many places that uzbl is better of the other browsers to show the videos because it shows the video trough omxplayer, and omxplayer alone is really great! instead all the browsers by themselves are very weak showing ,p4 videos into raspberry/raspbian 8.. 
Is there a way to say to uzbl that omxplayer is a plugin for the videos? it looks ot be updated at the last version at least.
how to compile the downloaded ubl files? (I have them into home/pi/uzbl)

**Comment by matteoraggi** [comment]
in the uzbl folder, 
`git brunch` says 
* master
and `git pull` says
 already up-to-date
but the raspbian repository says that I am using the ancient version of 14 may 2012..

**Comment by matteoraggi** [comment]
about KDE WebKit Library, the add/rmove software of raspbian 8.0 says: 
libkdewebkit5-4:4.14.2-5+deb8u1

**Comment by keis** [comment]
When installing from git you should not have uzbl installed from debian repositories at the same time and you will not be able to trust that version. You build and install by running `make PREFIX=/usr` followed by `make install PREFIX=/usr`. Make sure you don't get any errors from running that from missing headers or something else.

There is no specific integration with omxplayer in uzbl so the way you would connect them is writing a script for uzbl (reacting to some event or a key command) that launches omxplayer

**Comment by matteoraggi** [comment]
thanks! I uninstalled uzbl from the debian repositories, then I was following these instructions:
https://www.uzbl.org/wiki/ubuntu
but when I am at step 3, writing MAKE, then I receive this answer:
`Requested 'gio-2.0 >= 2.44' but version of GIO is 2.42.1
Requested 'gio-2.0 >= 2.44' but version of GIO is 2.42.1
cc -std=c99  -ggdb -W -Wall -Wextra -pthread -Wunused-function -D_XOPEN_SOURCE=500 -DARCH=\"armv7l\" -DCOMMIT=\"v0.9.1-2-gd603e8b\" -DLIBDIR=\"/usr/local/lib/uzbl\" -DHAVE_LIBSOUP_CHECK_VERSION  -c -o src/uzbl-core.o src/uzbl-core.c
In file included from src/uzbl-core.h:4:0,
                 from src/uzbl-core.c:30:
src/webkit.h:4:27: fatal error: webkit/webkit.h: File o directory non esistente
 #include <webkit/webkit.h>
                           ^
compilation terminated.
<incorporato>: set di istruzioni per l'obiettivo "src/uzbl-core.o" non riuscito
make: *** [src/uzbl-core.o] Errore 1`

**Comment by keis** [comment]
Oh right we've had this come up before (#281) we require a newer version of glib-gio than what raspbian stable carries.

**Comment by matteoraggi** [comment]
yeah! thanks the help of a friend I installed glib-gio 2.5 and it require gettext and it require libffilibmount.
now it works like the older version of 4 years and half ago, but it always show a ssl error, number 7771 if I don't go to a url and 8078 if I og to www.google.it (and it works!)
```
(uzbl-core:7771): GLib-GObject-WARNING **: g_object_set_valist: object class 'GtkLabel' has no property named 'margin'

(uzbl-core:7771): GLib-GObject-WARNING **: g_object_set_valist: object class 'GtkLabel' has no property named 'margin'

(uzbl-core:7771): libsoup-WARNING **: Could not set SSL credentials from '= /etc/ssl/certs/ca-certificates.crt': Apertura del file "/home/pi/= /etc/ssl/certs/ca-certificates.crt" non riuscita: File o directory non esistente
** Message: PROP_TIMELINE_PROFILING_ENABLED has been deprecated
```
 if I try to go to  : https://www.eforte.it , then it says a different error and it doesn't open the page, the same happen to https://www.eforte.eu when instead normally the other browser open it.

**Comment by matteoraggi** [comment]
about the trouble to surf into some ssl websites, I tryed to export the certificate from chromium into this path  /etc/ssl/certs/ and then run` update-ca-certificates` but it answer:
`Updating certificates in  /etc/ssl/certs/... rm: cannot remove 'ca-certificates.crt': Permission denied`
